src/pages/meditation/components/MeditationEndButton.tsx

```diff
@@ -0,0 +1,35 @@
+import styled from '@emotion/styled';
+import Button from '@components/Button';
+import { atom, useRecoilState } from 'recoil';
+
+const EndButtonContainer = styled.div`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin-top: 50px;
+`;
+
+export const endButtonPushed = atom({
+  key: 'endButtonPushed',
+  default: false
+});
+
+const MeditationEndButton = () => {
+  const [_, setPushed] = useRecoilState(endButtonPushed);
```

Comment:
set함수만 쓰시려면 이렇게 할 수 있을거 같아요
```suggestion
  const setPushed = useSetRecoilState(endButtonPushed);
```